### PR TITLE
Feature/review component

### PR DESCRIPTION
**Pull request by userigorgithub**
# Description
Please include a summary of the changes and/or which issue is fixed. Please also include relevant motivation and context. 
-Create Review.tsx and Review.css files and pass information to display on page through ReviewForm.tsx
-Create POST to display data in page and server

Were there any issues that arose? 

Is there anything you need from your teammates? 
Amazing job all!

Any other comments, questions, or concerns?

## Issues
Closes #

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?
- [x] React Dev Tools
- [x] Functionality check through web browser
- [ ] Ran Cypress test suite

# Checklist:
- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..).
- [x] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have deleted all comments after functionality and understanding is final.
- [ ] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers.

### CODE REVIEW

File: src/Components/ReviewForm.tsx [hunk 1]
```diff
@@ -1,56 +1,109 @@
...
+        method: 'POST',
+        body: JSON.stringify({
+          "id": this.props.id,
+          "name": this.state.name,
+          "review": this.state.review
+        }),
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      })
```
Comment (rachellhill) [inline src/Components/ReviewForm.tsx:69]: Successful POST! Great work!

File: src/Components/ReviewForm.tsx [hunk 2]
```diff
@@ -1,56 +1,109 @@
...
+        } else {
+          throw Error(response.statusText)
+        }
+      })
+      .then(data => {
+        this.setState({ reviews: [...this.state.reviews, {
+        id: this.props.id,
+        name: this.state.name,
+        review: this.state.review
+       }] })
```
Comment (rachellhill) [inline src/Components/ReviewForm.tsx:84]: great work with this!

**Review by rachellhill — APPROVED** [review]
Awesome job with POST

File: src/Components/ReviewForm.tsx [hunk 3]
```diff
@@ -1,56 +1,109 @@
...
+      })
+      .catch(error => console.log('errrrror')
+      )
```
Comment (stephanie-roe) [inline src/Components/ReviewForm.tsx:87]: great posting here!

File: src/Components/App.tsx [hunk 4]
```diff
@@ -4,7 +4,8 @@ import Breweries from './Breweries';
 import Brewery from './Brewery';
 import BreweryDetails from './BreweryDetails'
 import NavBar from './NavBar';
-import { Route, RouteComponentProps } from 'react-router-dom'
+import { Route, RouteComponentProps } from 'react-router-dom';
+import ReviewForm from './ReviewForm';
 // import { threadId } from 'worker_threads';
```
Comment (aominhlong) [inline src/Components/App.tsx:9]: We can remove unused code from our app

File: src/Components/ReviewForm.tsx [hunk 5]
```diff
@@ -1,56 +1,109 @@
...
     name: string,
-    content: string
+    review: string
 }
 
 class ReviewForm extends React.Component<ReviewFormProps, ReviewFormState> {
     state: ReviewFormState = {
-        name: "",
-        content: "",
+        id: this.props.id,
```
Comment (stephanie-roe) [inline src/Components/ReviewForm.tsx:25]: I think we can remove this!

**Review by aominhlong — APPROVED** [review]

**Review by stephanie-roe — APPROVED** [review]
Looking good! This is pretty much our MVP

**Comment by aominhlong** [comment]
Great work! We just need to make sure to remove any commented out code or console logs for the final project.

**Pull request merged**